.gitattributes

```diff
@@ -0,0 +1,3 @@
+/pulldown-cmark/third_party/** eol=lf
+/pulldown-cmark/specs/** eol=lf
+/pulldown-cmark/tests/suite/*.rs eol=lf
```

Comment:
This isn't really necessary for the Rust source files: https://github.com/rust-lang/rust/issues/62865

Reply:
Thanks for the point. I added this line since Git warned that files' line-endings will be overwritten by Git automatically. But I think it is just an issue of my local repository's configuration and should not be solved by this settings file. I'll remove it.